homeassistant/components/alert.py

```diff
@@ -230,6 +234,9 @@ def end_alerting(self):
         self._cancel()
         self._ack = False
         self._firing = False
+        if self._done_message != '' and self._send_done_message:
```

Comment:
If that will be default None, you can only use `if self._done_message and self._send_done...`